Initium-Odp/war/odp/javascript/script.js

```diff
@@ -1009,21 +1008,24 @@ function reloadPagePopup(quietly)
 {
 	if (currentPopupStackIndex==0)
 		return;
-	
+
 	var pagePopupId = "page-popup"+currentPopupStackIndex;
 	var content = $("#"+pagePopupId+"-content");
-	
+	var iframe = content.find("#"+pagePopupId+"-iframe");
+	if (iframe!=null)
+		content = iframe;
+
 	var url = content.attr("src");
+
+	if (quietly)
```

Comment:
Sorry I might have missed this from your previous commit, so I assume that this has been a problem throughout... If I'm reading it right.